content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -49,7 +49,7 @@ Analyze the current live cluster, simulating the effect of applying additional y
 $ istioctl analyze @samples/bookinfo/networking/bookinfo-gateway.yaml@ @samples/bookinfo/networking/destination-rule-all.yaml@
 Error [IST0101] (VirtualService bookinfo.default samples/bookinfo/networking/bookinfo-gateway.yaml:39) Referenced host not found: "productpage"
 Error: Analyzers found issues when analyzing namespace: default.
-See https://istio.io/v1.9/docs/reference/config/analysis for more information about causes and resolutions.
+See https://istio.io/v1.10/docs/reference/config/analysis for more information about causes and resolutions.
```

Comment:
Should we use `{{< istio_version >}}`, so this doesn't need to change every release?
```suggestion
See https://istio.io/v{{< istio_version >}}/docs/reference/config/analysis for more information about causes and resolutions.
```

Reply:
@frankbu I had thought of that as a quick fix, but we don't expand it in the snips.sh. I didn't look at the snips.sh creation to see if we could do the expansion first. The other thought I had was to update the test to verify the portions before and after the version. In the end, I decided that updating the document was the easiest.

Reply:
I see. You might want to make the third suggested change anyway (the one that's not in a snip), so at least we don't have to keep updating that part.

Reply:
Fixed.